python/src/lib.rs

```diff
@@ -197,6 +204,20 @@ impl RawDeltaTable {
             .map_err(PyDeltaTableError::from_raw)
     }
 
+    // Run the History command on the Delta Table: Returns provenance information, including the operation, user, and so on, for each write to a table.
+    pub fn history(&mut self, limit: Option<usize>) -> PyResult<Vec<RawDeltaTableCommitInfo>> {
+        let history = self
+            ._table
+            .history(limit)
+            .map_err(PyDeltaTableError::from_raw)?;
+        Ok(history
+            .iter()
+            .map(|c| RawDeltaTableCommitInfo {
```

Comment:
I don't see a lot of value having a `RawDeltaTableCommitInfo` struct here just wrapping a JSON string. Perhaps we could just return a vector of serialized string here to simplify the code? In the long run, we can also update this method to return a vector of python dictionary to avoid the extra round of JSON parsing in python.

Reply:
👍 I agree. 
For now, we can parse the JSON string on the Python side even if it adds overhead. Implementing a direct conversion for a reasonable subset of Python's built-in types to serde_json::Value in PyO3 is a non-trivial amount of work.